flocker/acceptance/test_api.py

```diff
@@ -0,0 +1,96 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Tests for the control service REST API.
+"""
+
```

Comment:
`from __future__ import unicode_literal`, maybe?  The code does some text formatting on bytes as written now.